### PR TITLE
Add API to delete aid packages

### PR DESCRIPTION
**Pull request by udda1996**
fixes https://github.com/LSFLK/MedicinesforLK/issues/210

### CODE REVIEW

File: service.bal [hunk 1]
```diff
@@ -112,6 +112,12 @@ service /donor on new http:Listener(9090) {
         check resultStream.close();
         return aidPackageUpdates;
     }
+
+    # A resource for deleting an aidPackage
+    resource function delete [string donorID]/aidpackages/[int AidPackageID]() returns error? {
+        sql:ExecutionResult _ = check dbClient->execute(`DELETE FROM AID_PACKAGE_ITEM PACKAGEID=${AidPackageID}`);
```
Comment (ayeshLK) [inline service.bal:118]:
```suggestion
        sql:ExecutionResult _ = check dbClient->execute(`DELETE FROM AID_PACKAGE_ITEM WHERE PACKAGEID=${AidPackageID}`);
```

File: service.bal [hunk 2]
```diff
@@ -112,6 +112,12 @@ service /donor on new http:Listener(9090) {
...
+        sql:ExecutionResult _ = check dbClient->execute(`DELETE FROM AID_PACKAGE WHERE PACKAGEID=${AidPackageID}`);
```
Comment (ayeshLK) [inline service.bal:119]: I think we need to wrap these dp updates in a transaction.

File: service.bal [hunk 3]
```diff
@@ -112,6 +112,23 @@ service /donor on new http:Listener(9090) {
...
+    resource function delete [string donorID]/aidpackages/[int AidPackageID]() returns error? {
+        AidPackageItem[] aidPackageItems = [];
+        stream<AidPackageItem, error?> resultStream = dbClient->query(`SELECT PACKAGEID,QUOTATIONID,NEEDID,QUANTITY,TOTALAMOUNT FROM AID_PACKAGE_ITEM  
+                                               WHERE PACKAGEID=${AidPackageID};`);
+        check from AidPackageItem aidPackageItem in resultStream
+            do {
+                aidPackageItems.push(aidPackageItem);
+            };
+        check resultStream.close();
+        foreach AidPackageItem aidPackageItem in aidPackageItems {
```
Comment (anupama-pathirage) [inline service.bal:126]: We need to do these updates and deletes in a transaction block?

**Review by anupama-pathirage — APPROVED** [review]

**Comment by jayasanka-sack** [comment]
I think this change should be on the Admin API, not here.

**Pull request merged**